boards/mro/sitl/default.px4board

```diff
@@ -0,0 +1,75 @@
+CONFIG_PLATFORM_POSIX=y
```

Comment:
Did you copy the sitl target as the basis for these files? What had to be changed?

Reply:
Yup, it was too much to go the other way around. Not much needed to be changed due to the way sitl has to "fake" a lot of the IO.